### PR TITLE
MAID-3120: docs/license: update LICENSE details

### PR DESCRIPTION
**Pull request by pravarseth**
made license text in readme more consitent with other repos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 ## License
 
-This is dual-licensed under the Modified BSD ([LICENSE-BSD](LICENSE-BSD) https://opensource.org/licenses/BSD-3-Clause) or the MIT license ([LICENSE-MIT](LICENSE-MIT) https://opensource.org/licenses/MIT) at your option.
+This SAFE Network library is dual-licensed under the Modified BSD ([LICENSE-BSD](LICENSE-BSD) https://opensource.org/licenses/BSD-3-Clause) or the MIT license ([LICENSE-MIT](LICENSE-MIT) https://opensource.org/licenses/MIT) at your option.
```
Comment (bochaco) [inline README.md]: @pravarseth , shall we consider changing it for cases like this one which is not a library but a SAFE Network application?

Comment (nadiaburborough) [reply README.md]: @bochaco so change it to: "This SAFE Network `application` is dual-licensed.."?

Comment (bochaco) [reply README.md]: yes @nadiaburborough , I think it will be more accurate for sure.

**Comment by bochaco** [comment]
I'll be sending a PR with this change included since this has been inactive for long now.

**Pull request closed without merging**